apps/www/app/page.tsx

```
@@ -1,14 +1,26 @@
-    const worker = new Worker("/browser/opfs.worker.js");
-
-    worker.postMessage({ type: "LIST" });
-    worker.onerror = console.log;
-    worker.onmessage = console.log;
-  }, []);
-  return null;
+  const opfs_worker = useOPFSWorker();
+
+  useEffect(
+    function () {
+      if (!!!opfs_worker) return;
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)